src/coreclr/scripts/superpmi.py

```diff
@@ -2964,21 +2989,22 @@ def write_pivot_section(row):
             write_pivot_section("Overall")
             write_pivot_section("MinOpts")
             write_pivot_section("FullOpts")
-        else:
+        elif include_details:
```

Comment:
Doesn't this need to remain an "else"?

Reply:
I guess you don't want to display anything when there are no diffs, which makes sense. There is no "header" here?

Reply:
fixed.